show-device-location-using-indoor-positioning/src/main/java/com/esri/arcgismaps/sample/showdevicelocationusingindoorpositioning/MainActivity.kt

```diff
@@ -142,10 +142,8 @@ class MainActivity : AppCompatActivity() {
             // load each FeatureTable
             featureTable.load().onSuccess {
                 // IPS_Positioning table needs to be present
-                if (featureTable.tableName == "IPS_Positioning") {
+                if (featureTable.tableName == "ips_positioning") {
```

Comment:
The table name works for Building M service I used to test
https://runtimecoretest.maps.arcgis.com/home/item.html?id=41fd5a7ee7044a40ad73f7f4a79a5a25

Reply:
even after I get the above error, and keep the app running for some time ~10 mins
I see the GNSS signals being detected.
Spooky behavior I say!